guides/integrate-firebase.md

```diff
@@ -0,0 +1,347 @@
+- How to authenticate with a passkey
+- How to configure the OpenID Connect plugin
+
+## Prerequisites
+
+- Set up a [developer account](/docs/v1/workflows/account-setup)
+
+## Firebase
+
+- [Authenticate Using OpenID Connect on Android](https://firebase.google.com/docs/auth/android/openid-connect)
+- [Authenticate Using OpenID Connect on Apple platforms](https://firebase.google.com/docs/auth/ios/openid-connect)
+- [Authenticate Using OpenID Connect in web apps](https://firebase.google.com/docs/auth/web/openid-connect).
+
+## Install the Embedded SDK
+
+In order to use Beyond Identity functionality in your application, you will need to install the Beyond Identity SDK. The [Android SDK](https://github.com/gobeyondidentity/bi-sdk-android), the [Swift SDK](https://github.com/gobeyondidentity/bi-sdk-swift), and the [JavaScript SDK](https://github.com/gobeyondidentity/bi-sdk-js) provides functionality from passkey creation to passwordless authentication. A set of functions are provided to you through an `Embedded` namespace.
+
+## Initialize the Embedded SDK
+
+Once you've installed the SDK, initialize it so that you can call the Embedded functions.
+
+<Tabs groupId="platform" queryString>
+<TabItem value="android" label="Android">
+
+<SetupKotlin />
+
+</TabItem>
+<TabItem value="ios" label="iOS">
+
+<SetupSwift />
+
+</TabItem>
+<TabItem value="web" label="Web">
+
+<SetupJavaScript />
+
+</TabItem>
+</Tabs>
+
+## Set up Beyond Identity as an Identity Provider
+
+To set up Beyond Identity as an Identity Provider, you need to create a [Realm](/docs/v1/workflows/realms) to hold identities and configuration. Inside that realm, you'll need to create an [Application](/docs/v1/workflows/applications) that contains the authentication flow configuration. These can be configured in you admin console that was created for you when you signed up for a developer account.
+
+### Create a Realm
+
+From the admin console, click the realm drop-down, and click **Create new realm**.
+
+<Arcade clip={Clip.CreateRealm} />
+
+### Create an Application
+
+From the admin console, click **Apps**, then click **Add app**.
+
+There is a lot to configure when creating an application. When creating your application make sure:
+
+- Redirect URIs contains your application's App Scheme or Universal URL
+- Configuration Type (in the Authenticator Config tab) is set to `Embedded SDK`
+- Invoke URL (in the Authenticator Config tab) contains your application's App Scheme or Universal URL
+- Invocation Type (in the Authenticator Config tab) is set to `Automatic`.
+
+For help choosing options, visit the following guides:
+
+- [Workflows: Applications](/docs/v1/workflows/applications)
+- [Authenticator Config](/docs/v1/platform-overview/authenticator-config)
+
+<Arcade clip={Clip.CreateApplication} />
+
+Now that we have created our application, we are ready to configure our custom OpenID Connect provider in the [Firebase console](https://console.firebase.google.com).
```

Comment:
This part is easily missed after the arcade. Maybe its own section will help it stick out?

```suggestion
### Configure OpenID Connect Provider
Now that we have created our application, we are ready to configure our custom OpenID Connect provider in the [Firebase console](https://console.firebase.google.com).
```

Reply:
Agreed.